src/server/misc/cmds/cmds_non1.20.go

```diff
@@ -0,0 +1,9 @@
+//go:build !go1.20
+
+package cmds
+
+import (
+	"net"
+)
+
+func setupControl(d *net.Dialer) {}
```

Comment:
Long term, we aren't going to have both of these?  We only build with one version of Go, so it's going to be that version.  It seems like we'll also call the noop setupControl for versions of go > 1.20.

Reply:
Yeah.  2.5.x is go 1.19 (which introduced the ControlContext function) and 2.6.x+ is go 1.20.  I would like to backport this and get the tools into 2.5.1, but without printing the fd # of the created socket (which I thought was nifty and helps if looking at strace or something, but I only added without knowing that you couldn't do it in 1.19 :))